static/src/javascripts/projects/journalism/modules/get-campaign.js

```diff
@@ -1,27 +1,18 @@
 // @flow
 import config from 'lib/config';
 
-export const getCampaign = () => {
+export const getCampaigns = () => {
     // eslint-disable-next-line
     const isCallout = campaign => campaign.fields._type === 'callout';
     const allCampaigns = config.get('page.campaigns');
-
-    // take the last added campaign as campaign to show
     const allCallouts = allCampaigns.filter(isCallout);
-    const campaignToShow = allCallouts[allCallouts.length - 1];
 
-    if (campaignToShow) {
-        const {
-            callout,
-            description,
-            formFields,
-            formId,
-        } = campaignToShow.fields;
-        return {
-            title: callout,
-            description: description || `<p>&nbsp;</p>`,
-            formFields,
-            formId,
-        };
-    }
+    return allCallouts.map(callout => ({
+        name: callout.name,
+        title: callout.fields.callout,
+        description: callout.fields.description || `<p>&nbsp;</p>`,
```

Comment:
yes fair point. The line height is needed by the design, but I can see if I can add that into the template rather than mixing it in here. Simply returning an empty string makes more sense from a data perspective

Reply:
Have added a min-height in the css